### PR TITLE
[8.x.x] Updated flat style

### PR DESCRIPTION
**Pull request by patricia-martinez-imatia**
- Updated flat styles
- Added the ability to configure status colors(primary, success, danger,warning, info,..)

### CODE REVIEW

**Review by pacogomezimatia — APPROVED** [review]

**Review by manuel-cid — APPROVED** [review]

File: src/styles/ontimize-theme-styles-flat.scss [hunk 1]
```diff
@@ -1,171 +1,105 @@
...
-    }
 
-    &.o-button-info {
+    &.o-button-danger,
+    &.o-button-default,
+    &.o-button-info,
+    &.o-button-primary,
+    &.o-button-success {
       button {
         color: white;
```
Comment (daniel-grana) [inline src/styles/ontimize-theme-styles-flat.scss:80]: Could this color 'white' cause problems with dark-mode?

File: src/styles/ontimize-theme-styles-flat.scss [hunk 2]
```diff
@@ -174,30 +108,29 @@
...
 
   .ontimize-table.o-table {
     .o-table-button {
       button.mat-stroked-button {
         border: 0;
-        border-radius: 0;
+      }
 
+      button {
+        color: white;
```
Comment (daniel-grana) [inline src/styles/ontimize-theme-styles-flat.scss:120]: The same as previous comment

**Review by daniel-grana — CHANGES_REQUESTED** [review]

File: src/styles/ontimize-theme-styles-flat.scss (hunk 1 above)
Comment (patricia-martinez-imatia) [reply src/styles/ontimize-theme-styles-flat.scss:80]: It also will be white

File: src/styles/ontimize-theme-styles-flat.scss (hunk 2 above)
Comment (patricia-martinez-imatia) [reply src/styles/ontimize-theme-styles-flat.scss:120]: It also will be white

**Review by daniel-grana — APPROVED** [review]

**Pull request merged**